src/vmm/src/logger/logging.rs

```diff
@@ -242,13 +242,13 @@ pub struct LevelFilterFromStrError(String);
 impl FromStr for LevelFilter {
     type Err = LevelFilterFromStrError;
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "Off" | "OFF" => Ok(Self::Off),
-            "Trace" | "TRACE" => Ok(Self::Trace),
-            "Debug" | "DEBUG" => Ok(Self::Debug),
-            "Info" | "INFO" => Ok(Self::Info),
-            "Warn" | "WARN" | "Warning" | "WARNING" => Ok(Self::Warn),
-            "Error" | "ERROR" => Ok(Self::Error),
+        match s.to_ascii_lowercase().as_str() {
```

Comment:
You can add a `Fixes: 9280cc25c72a096c1f3a09083037ff6af02b16ef` trailer to this commit.

Reply:
added a link to merged commit.